README.md

```diff
@@ -11,8 +11,8 @@ _Read more about Dangerzone in the blog post [Dangerzone: Working With Suspiciou
 
 ## Getting started
 
-- Download [Dangerzone 0.3.2 for Mac](https://github.com/firstlookmedia/dangerzone/releases/download/v0.3.2/Dangerzone-0.3.2.dmg)
-- Download [Dangerzone 0.3.2 for Windows](https://github.com/firstlookmedia/dangerzone/releases/download/v0.3.2/Dangerzone-0.3.2.msi)
+- Download [Dangerzone 0.4.0 for Mac](https://github.com/firstlookmedia/dangerzone/releases/download/v0.4.0/Dangerzone-0.4.0.dmg)
+- Download [Dangerzone 0.4.0 for Windows](https://github.com/firstlookmedia/dangerzone/releases/download/v0.4.0/Dangerzone-0.4.0.msi)
```

Comment:
FWIW these URLs just redirect to the `freedomofpress` repo so I think we could update them without issue

Reply:
You're right Erik. I added a commit on top that replaces these references, and some other ones I found in the repo.